tutorials-v4/brmesolve-cavity-QED.md

```diff
@@ -0,0 +1,184 @@
+    [result_me_strong, result_brme_strong],
+    ylabels=["<n_cav>", "<n_atom>"],
+    show_legend=True,
+);
+```
+
+In the strong coupling regime there are differences in the solution of the two solvers. This is due to the fact that the eigenstates of the system with strong coupling have contributions from atom and the cavity system, i.e. hybridized eigenstates. The Lindbladian master equation solver assumes that the dissipation of a subsystem, here losses in the cavity, does not affect the state of the other subsystem, here the atom. However, for the strongly coupled Hamiltonian this is not true anymore and the dissipation leads to transitions between eigenstates of the coupled system, i.e. also affects the atomic state. By definition the Bloch-Redfield solver takes this hybridization into account and yields the more accurate result for such systems.
+
+We can see this hybridization of the eigenstates from the energy levels. In the system without any interaction, both the atom and the cavity have the same energy levels as we defined them with the similar frequency `w_0`.
+For weak interaction the energy levels split up by a small constant, but do not mix up. For strong coupling the energy levels from originally different states bunch together and show the mixing contribution of atom and cavity to the systems eigenstates.
+
+```python
+plot_energy_levels([H_no, H_weak, H_strong],
+                   labels=["no coupling", "weak", "strong"]);
+```
+
+### Non-secular solution
+The `qutip.brmesolve()` function automatically uses the secular approximation, i.e. assumes that all fast-oscillating terms in the Hamiltonian can be neglected. However, this approximation is not necessary for the Bloch-Redfield solver and this option can be deactived by setting `use_secular=False` in the arguments. This setting can be useful in some simulations. For the above example of strongly coupled atom-cavity system, dropping the approximation does not change the solution by much.
+
+```python
+result_brme_nonsec = brmesolve(H_strong, psi0, times, a_ops,
+                               e_ops, use_secular=False)
+plot_expectation_values([result_brme_strong, result_brme_nonsec],
+                        ylabels=["<n_cav>", "<n_atom>"], show_legend=True);
+```
+
+### Trace of states
+The Lindbladian Master Equation approach guarantees that the density matrix follows a physical evolution, i.e. the evolution is trace and positivity preserving (up to numerical precision). 
+
+A down-side of the Bloch-Redfield solver is that it does not guarantee this physical evolution. Hence the trace of the resulting density matrix can vary. Below we plot the trace of the density matrices for the evolution of the weakly coupled system. The trace of the Bloch-Redfield density matrices slightly deviates from the expected trace $1$ (here: in the range of $10^{-12}$ ). This small deviation was not a problem for the simulation above, but can become a problem for other systems.  
```

Comment:
The rendering from matplotlib is a bit weird here, I agree. The top left `1e-12+10e-1` indicates the scale and shift of the y-axis. In this case the y-axis is shifted by `10e-1` (i.e. 1) and the scaled by `1e-12`. I guess this is too avoid writing `1e-12` at each tick on the y-axis.

Reply:
Perhaps let us just leave a comment about this strange rendering too? It's quite odd to me that it happens automatically? Otherwise maybe we could set the limits of the y-axis ourselves?

Reply:
Setting the limits ourselves results in the same plot. I think this is mainly to avoid weird ticks on the y-label. 
I have added a comment on the plot's y-labels now.